voicebox/node.py

```diff
@@ -150,5 +197,7 @@ def end_call(self, addr: str, inform_connection: bool = True):
     @staticmethod
     def validate_connection(address: str) -> bool:
 
-        return bool(input("Would you like to connect to this client at {}? ".format(address)))
+        return True

```

Comment:
The `validate_connection` method currently returns `True` unconditionally, which means any connection is accepted. This could be a potential security risk. Consider implementing a proper validation mechanism. [important]